libraries/Microsoft.Bot.Schema/Teams/ConversationList.cs

```diff
@@ -27,18 +27,16 @@ public ConversationList()
         /// <param name="conversations">The IList of conversations.</param>
         public ConversationList(IList<ChannelInfo> conversations = default)
         {
-            Conversations = conversations;
+            Conversations = conversations ?? new List<ChannelInfo>();
```

Comment:
Can we use a ??= operator here to not have to create another List<ChannelInfo>() since it was already initialized in the property declaration?